pkg/tuple/tuple.go

```diff
@@ -63,6 +63,19 @@ func GetObjectRelationAsString(objectRelation *openfgapb.ObjectRelation) string
 	return objectRelation.GetObject()
 }
 
+func GetRelationReferenceAsString(rr *openfgapb.RelationReference) string {
+	if rr == nil {
+		return ""
+	}
+	if _, ok := rr.RelationOrWildcard.(*openfgapb.RelationReference_Relation); ok {
```

Comment:
Nested if/else is redundant. More elegantly written as

```
if _, ok := ...; ok {
    return ...
}

if _, ok := ...; ok {
    return ...
}

panic("unexpected relation reference")
```

Please add a test for this in `pkg/tuple/tuple_test.go`.